tasks/main.yml

```diff
@@ -113,28 +113,54 @@
     name: "{{ camac_releasedir }}/camac/django/"
     extra_args: "--upgrade"
 
+- name: reset django_migrations table
+  shell: |
+    psql \
+        -h {{ camac_db_host }} \
+        -p {{ camac_db_port }} \
+        -U {{ camac_db_username }} \
+        -d {{ camac_db_dbname }} \
+        -c "TRUNCATE django_migrations"
+  when: camac_db_run_fake_migrations and camac_db_fake_migrations is defined
+  environment:
+    PGPASSWORD: "{{ camac_db_password }}"
+
+- name: run camac fake migrations
+  shell: "python3 {{ camac_releasedir }}/camac/django/manage.py migrate --fake {{ item.app }} {{ item.migration }}"
+  when: camac_db_run_fake_migrations and camac_db_fake_migrations is defined
+  environment:
+    DJANGO_SETTINGS_MODULE: "camac.settings"
+    DJANGO_DATABASE_NAME: "{{ camac_db_dbname }}"
+    DJANGO_DATABASE_USER: "{{ camac_db_username }}"
+    DJANGO_DATABASE_PASSWORD: "{{ camac_db_password }}"
+    DJANGO_DATABASE_HOST: "{{ camac_db_host }}"
+    DJANGO_DATABASE_PORT: "{{ camac_db_port }}"
+    APPLICATION: "{{ camac_application }}"
+  loop: "{{ camac_db_fake_migrations }}"
+
+
 - name: run camac migrations
-  shell: django-admin migrate
+  shell: "python3 {{ camac_releasedir }}/camac/django/manage.py migrate"
```

Comment:
You'll also need to ensure that py3 gets installed beforehand, this is usually done at @ $CUSTOMER using their existing process.

Reply:
:+1: